llvm/docs/FatLTO.rst

```diff
@@ -76,3 +76,56 @@ Supported File Formats
 The current implementation only supports ELF files. At time of writing, it is
 unclear if it will be useful to support other object file formats like ``COFF``
 or ``Mach-O``.
+
+Usage
+===========
+
+Users are expected to pass ``-ffat-lto-objects`` to clang in addition to one of
+the ``-flto`` variants. Without the ``-flto`` flag, ``-ffat-lto-objects`` has
+no effect.
+
+
+Compile and link. Use the object code from the fat object without LTO.
```

Comment:
I'm not sure these "compile and link" examples make a lot of sense. I think fat LTO is only meaningful if compile and link is separate.

Reply:
Yeah, I'll revise the examples to have a common first compile step and then different link steps.